OOTPISP_L1/Bases/FLine.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace OOTPISP_L1
+{
+    public class FLine
```

Comment:
What is F in your name?

Reply:
It's FigureLine to resolve name conflict between my Line class and System.Windows.Shapes Line